site/src/App/Markdown/Markdown.css.ts

```diff
@@ -0,0 +1,31 @@
+import { style, composeStyles } from '@vanilla-extract/css';
```

Comment:
Don't need to use `composeStyles` anymore.

Reply:
Will update this separately with a PR that bumps vanilla and rolls it through.